src/components/Header/Header.scss

```diff
@@ -0,0 +1,125 @@
+@use "../../styles/reset";
+@use "../../styles/variables" as *;
+@use "../../styles/typography";
+
+
+.header{
+    padding: 1rem;
+    background-color: $secondary-color;
+    &__image-div{
+        float: right;
+        width: 2.2rem;
+        height: auto;
+        &--image{
+            border-radius: 50%;
+            background-color: $white-color;
+        }
+    }
+   &__title-div{
+    &--title{
+        color: $primary-color;
+        margin-bottom: unset;
+        font-family: $font-family-header-title;
+    }
+    &--subtitle{
+        margin-top: unset;
+        font-family: $font-family-modern-header;
+    }
+   }
+   
+}
+
+
+@media screen and (min-width: $tablet-breakpoint) {
+    .header{
+        padding: 3rem;
+        &__image-div{
+            width: 3.5rem;
+            
+        }
+       &__title-div{
+        &--title{
+            font-size: 2rem;
+        }
+        &--subtitle{
+            font-size: 4.5rem;
+        }
+       }
+    }
+    
+}
+
+  @media screen and (min-width: $desktop-breakpoint) {
+    .header{
+        padding: 4rem;
+        &__image-div{
+            width: 4rem;
+            
+        }
+       &__title-div{
+        &--title{
+            font-size: 2.5rem;
+        }
+        &--subtitle{
+            font-size: 5.5rem;
+        }
+       }
+    }
+  }
+
+
+  @media screen and (min-width: $large-desktop-breakpoint) {
+    .header{
+        padding: 5rem;
+        &__image-div{
+            width: 5rem;
+            
+        }
+       &__title-div{
+        &--title{
+            font-size: 3.5rem;
+        }
+        &--subtitle{
+            font-size: 7.5rem;
+        }
+       }
+    }
+  }
+
+
+  @media screen and (min-width: $large-desktop-breakpoint) {
+    .header{
+        padding: 6rem;
+        &__image-div{
+            width: 6.5rem;
+            
+        }
+       &__title-div{
+        &--title{
+            font-size: 4rem;
+        }
+        &--subtitle{
+            font-size: 8.5rem;
+        }
+       }
+    }
+  }  
+
+
+  @media screen and (min-width: $extra-large-desktop-breakpoint) {
+    .header{
+        padding: 8rem;
+        &__image-div{
+            width: 9.5rem;
+            
+        }
+       &__title-div{
+        &--title{
+            font-size: 6rem;
+        }
+        &--subtitle{
+            font-size: 11.5rem;
+        }
+       }
+    }
+  }
```

Comment:
The size of the header is far too large on screen sizes from tablet to desktop, ik we don't have wireframes to work from for larger screen sizes so just get this sized sensibly